node/tests/commands/destroy.spec.js

```diff
@@ -49,6 +50,26 @@ describe('destroy', () => {
     );
   });
 
+  describe('skipStateRefresh argument', () => {
+    it.each`
+      skipStateRefreshOption | existing
+      ${'true'}              | ${'true'}
+      ${'false'}             | ${'false'}
+      ${undefined}           | ${undefined}
```

Comment:
if they are the same, why not creating a single `options` and expect the function to be called with it (like in proper data test above)? why distinguish the passed parameter and the expected?

Reply:
Done: 36cdd6b6a2cb7d845c29dd480ff7dfa9516b16fd